vyxal/transpile.py

```diff
@@ -89,9 +89,17 @@ def transpile_token(
         # screws up escape sequences.
 
         # So instead, we have to manually escape the string
-        string = f"{string!r}"
-        string = re.sub(r"\\?\\([ntbrf])", r"\\\1", string).replace("\\`", "`")
-        return indent_str(f"stack.append({string})", indent)
+        temp = ""
+        string = collections.deque(string)
+        while string:
+            char = string.popleft()
+            if char == "\\" and string:
+                temp += "\\" + string.popleft()
+            elif char == '"':
+                temp += '\\"'
+            else:
+                temp += char
```

Comment:
Use an iterator instead:
```suggestion
        iterator = iter(string)
        for char in iterator:
            if char == "\\":
                temp += "\\" + next(iterator, "")
            elif char == '"':
                temp += '\\"'
            else:
                temp += char
```

Reply:
I still don't know what escape sequences you [think](https://github.com/Vyxal/Vyxal/commit/e6a4030cc649a0ba750b7904d0dc4539aacbe60e) `{string!r}` (which is the same as `{repr(string)}`) breaks.

Also, won't this reintroduce #253 and/or #359 and/or #379?

Reply:
If you have something like `\n` in a string, `{string!r}` changes it to `\\n`

Reply:
And it shouldn't reintroduce any of those issues - #253 was to do with the fact that the D flag used to be handled differently, #359 is because of the way raw strings work and #379 was because of the regex/substitution I tried to use.

Reply:
After some testing #253 was reintroduced. But that's fixed.